lib/reverse_sentence.rb

```diff
@@ -1,6 +1,28 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(1)
```

Comment:
Since you're doing `.scan` you end up with O(n) because it's making a new array.